src/Symfony/Component/AssetMapper/Tests/ImportMap/RemotePackageStorageTest.php

```diff
@@ -43,6 +43,10 @@ public function testGetStorageDir()
 
     public function testSaveThrowsWhenVendorDirectoryIsNotWritable()
     {
+        if ('\\' === \DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('chmod is not supported on Windows');
```

Comment:
Is this still necessary with the change done in the Filesystem component?

Reply:
I think i missed these changes... :/ 

KInda still think this is the main reason... but of course if ij the meantime something has been fixed on FS ... 😶‍🌫️ 

https://github.com/symfony/symfony/pull/57375/commits/e58b1021dbb1bc5b2f0abefc5a394387dd5bf020